python/pylibcudf/pylibcudf/column.pyx

```diff
@@ -767,6 +767,30 @@ cdef class Column:
             c_result = make_unique[column](self.view())
         return Column.from_libcudf(move(c_result))
 
+    cpdef size_type device_buffer_size(self):
```

Comment:
This should return a `uint64_t`, and accumulate into a `uint64_t`. Although columns are limited to `size_type` _rows_, that is not the limit on the number of _bytes_ they can contain.

```suggestion
    cpdef uint64_t device_buffer_size(self):
```